src/driver/types/CteCapabilities.ts

```diff
@@ -0,0 +1,16 @@
+export interface CteCapabilities {
+    /**
+     * Are RETURNING clauses supported in CTEs?
+     */
+    writable?: boolean;
+    /**
+     * Is RECURSIVE clause required for recursive CTEs?
+     */
+    requiresRecursiveHint?: boolean;
+
+    /**
+     * Is MATERIALIZED clause supported?
+     */
+
+    materializedHint?: boolean;
+}
```

Comment:
I think we need to fix formatting here.

Reply:
@pleerock As side note - maybe it is sensible to introduce prettier or similar tool to TypeORM development?